src/System.Text.Primitives/System/Text/Parsers/Utf16Parser_signed.cs

```diff
@@ -3,8 +3,9 @@
 // See the LICENSE file in the project root for more information.
 
 
-// Copyright (c) Microsoft. All rights reserved.
-// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+// Licensed to the .NET Foundation under one or more agreements.
```

Comment:
This text is duplicate. We should remove it.

Reply:
Good catch - I'll update the regex to fix this.